integrations/hermes/__init__.py

```
@@ -165,10 +243,27 @@ def _api(base: str, path: str, body: dict | None = None, method: str = "POST", s
+        _debug_log(
+            f"_api fail path={path} status={status} err={type(exc).__name__}: {exc} "
+            f"auth_set={bool(auth)} base={base} home={os.environ.get('HOME')!r} "
+            f"real_home={os.environ.get('HERMES_REAL_HOME')!r}"
+        )
         return None
 
 
+def _debug_log(msg: str) -> None:
+    """Best-effort local diagnose log (no secrets)."""
+    try:
+        log_path = Path("/tmp/agentmemory_plugin_debug.log")
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/agentmemory_plugin_debug.log"

(S108)